resources/Example.yamil

```diff
@@ -0,0 +1,48 @@
+def calculate(a: int, b: int) -> int = {
```

Comment:
Update the path to app directory, so move it inside without resources directory. Due to in Haskell folder structure app directory refers to the external files can be passed to the program, I mean connect with impure world.

Reply:
I don't understand your point of view where do you recommend me to move this Example ?